hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                </div>
+                <div class="modal-body" id="save-collection-warning-body">
+                    You have NO PERMISSION over the resource you are about to remove from this collection. Once confirmed, you CAN NOT add it back.
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal" id="save-collection-btn-cancel">
+                        Cancel
+                    </button>
+                    <button type="button" class="btn btn-danger" data-dismiss="modal" id="save-collection-btn-ok">
+                        Confirm
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {#    modal div: collection candidate table #}
+    <div class="modal fade" id="collection-candidate" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog" style="width:65%">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h4 class="modal-title" id="save-collection-warning-title">My Resources Pool</h4>
+                </div>
+                <div class="modal-body" id="save-collection-warning-body">
+                    <table id="collection-table-candidate" class="table-hover table-striped" width="100%">
+                        <thead>
+                        <tr>
+                            <th>Add</th>
+                            <th>Title</th>
+                            <th>Type</th>
+                            <th>Owners</th>
+                            <th>Sharing status</th>
+                            <th>My Permission</th>
+                            <th>Remove</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% if collection_candidate %}
+                            {% for res in collection_candidate %}
+                                <tr id="{{ res.short_id }}">
+                                    <td>
+                                        <input class="row-selector" type="checkbox" id="{{ res.short_id }}">
+                                    </td>
+                                    <td>
+                                        <strong><a href="{{ res.get_absolute_url }}" target="_blank">{{ res.metadata.title }}</a></strong></td>
+                                    <td>
+                                        {{ res.resource_type }}
+                                    </td>
+                                    <td>
+                                        {% for owner in res.raccess.owners.all %}
+                                            {% if forloop.counter0 > 0 %} · {% endif %}
```

Comment:
@zhiyuli Again, I think this empty if-endif block serves no purpose and can be removed.